app/javascript/Components/__tests__/create_tag_modal.test.jsx

```diff
@@ -21,14 +22,26 @@ describe("CreateTagModal", () => {
     props = {
       isOpen: true,
       onRequestClose: jest.fn().mockImplementation(() => (props.isOpen = false)),
-      grouping_id: 1,
```

Comment:
It's good to remove these props. In the other test files, please make sure you've also removed these props when adding in the context.